pages/common/gcloud-sql-backups-describe.md

```diff
@@ -0,0 +1,8 @@
+# gcloud sql backups describe
+
+> Retrieves information about a backup.
+> More information: <https://cloud.google.com/sdk/gcloud/reference/sql/backups/describe>.
+
+- Code description:
+
+`gcloud sql backups describe {{backup-id}} --instance={{instance-id}}{{GCLOUD_WIDE_FLAG …}}`
```

Comment:
```suggestion
`gcloud sql backups describe {{backup_id}} --instance={{instance_id}}`
```
We can use the snake case here and I don't think we need to specify the Gcloud-wide flag.